src/Forcings/Forcings.jl

```diff
@@ -1,13 +1,15 @@
 module Forcings
 
-export Forcing, ContinuousForcing, DiscreteForcing, Relaxation, GaussianMask, LinearTarget
+export Forcing, ContinuousForcing, DiscreteForcing, Relaxation, GaussianMask, LinearTarget, AdvectiveForcing
 
 using Oceananigans.Fields
 
 include("continuous_forcing.jl")
 include("discrete_forcing.jl")
 include("relaxation.jl")
+include("advective_forcing.jl")
 include("forcing.jl")
 include("model_forcing.jl")
+include("multiple_forcings.jl")
```

Comment:
Is that a reason for including files in this order? If not, should we have a criterium for choosing the order based on the programming style?

Reply:
I was asking because you didn't add the files at the end, but agree that shouldn't change anything.